app/controllers/actor_movies_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ActorMoviesController < ApplicationController
+  def create
+    ActorMovie.create(movie_id: params[:movie_id], actor_id: params[:actor_id])
```

Comment:
I recommend always using the `!` bang operator to make sure any errors that occur with `.create!` actually bubble up and we're able to use the related validation messages that come along with it. Think of `!` as "failing loudly".

Reply:
That said, I like that you created this controller for this user story and used the appropriate action!

Reply:
That's interesting! I _think_ Abdul recommended the opposite and recommends only using the bang operator in your tests, not your actions/methods.